binary_cps_format.md

```diff
@@ -0,0 +1,329 @@
+| name            | char[32]                                                                                                                      | display name for channel                                                                                                                                  |
+| descr           | char[32]                                                                                                                      | Description of the channel                                                                                                                                |
+| ch_location     | [geo_t](#geo_t-type-description)                                                                                              | transmitter location                                                                                                                                      |
+| infoblock       | [fmInfo_t](#fminfo_t-type-description), [dmrInfo_t](#dmrinfo_t-type-description), or [m17Info_t](#m17info_t-type-description) | Information block for the channel                                                                                                                         |
+
+#### Bandwidth lookup table
+
+| Bits | Bandwidth (kHz) |
+| ---- | --------------- |
+| 0b00 | 12.5            |
+| 0b01 | 20              |
+| 0b10 | 25              |
+| 0b11 | Reserved        |
+
+#### geo_t type description
+
+| Field       | Data Type | Description                                                                                                                                                        |
+| ----------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| ch_lat_int  | int8_t    | Floor of latitude (⌊x⌋; e.g. given latitude of 44.493889, this value is 44)                                                                                        |
+| ch_lat_dec  | uint16_t  | Fractional part of latitude, first rounded to the ten-thousandths, represented as a positive whole number (e.g. given latitude of 44.493889, this value is 4939)   |
+| ch_lon_int  | int8_t    | Floor of longitude (⌊x⌋; e.g. given longitude of 11.342778, this value is 11)                                                                                      |
+| ch_lon_dec  | uint16_t  | Fractional part of longitude, first rounded to the ten-thousandths, represented as a positive whole number (e.g. given longitude of 11.342778, this value is 3428) |
+| ch_altitude | uint16_t  | Altitude of the center of the radiatior of the transmitter, stored in meters MSL offset +500 (_what does this even mean, why_)                                     |
```

Comment:
@silseva maybe you can help answer why the codeplug proposal has a +500m offset for altitude? Is it because the dead sea is the lowest feasible location and that's ~-415m MSL?